app/src/main/java/ch/hikemate/app/ui/map/HikeDetailScreen.kt

```diff
@@ -200,6 +200,10 @@ fun HikeDetailScreen(
     profileViewModel.getProfileById(authViewModel.currentUser.value!!.uid)
   }
 
+  var wantToRunHike = false
+
+  LaunchedEffect(wantToRunHike) { navigationActions.navigateTo(Screen.RUN_HIKE) }
+
```

Comment:
Change: I'm not sure why you added this, but now the details screen always switches immediately to the run hike screen, because the `LaunchedEffect` will be executed once on the first composition. What was the goal?

Reply:
Yup, it should be fixed now, I made the launch effect check whether `wantToRunHike` is actually positive before it starts the navigation. Without this feature, spamming the run this hike button triggers multiple navigations to the run hike screen, which causes it to get launched multiple times...